src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DefaultSize = 10;
+    private static final String DefaultColor = "White";
+    private static final int Limit = 100;
+    private static final int TotalFigure = 5;
+    private Random random = new Random();
```

Comment:
```suggestion
    private static final int DEFAULT_SIZE = 10;
    private static final String DEFAULT_COLOR = "White";
    private static final int MAX_LIMIT = 100;
    private static final int TOTAL_FIGURE = 5;
    private Random random = new Random();
```

constants should be named in uppercase